docs/pages/management/guides/joining-nodes-azure.mdx

```diff
@@ -0,0 +1,399 @@
+
+  <Admonition
+    type="warning"
+    title="IMPORTANT"
+  >
+  As part of authentication, Teleport will be able to do anything allowed by the
+  identity. If you use a managed identity other than the one created in this
+  guide, we strongly recommend limiting its permissions and scope. 
+  </Admonition>
+</TabItem>
+<TabItem label="ARM Template">
+  Create a file named `teleport-create-identity.json` and copy the following into
+it:
+
+```json
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "metadata": {
+    "_generator": {
+      "name": "bicep",
+      "version": "0.5.6.12127",
+      "templateHash": "2227781763411200690"
+    }
+  },
+  "parameters": {
+    "roleName": {
+      "type": "string",
+      "defaultValue": "teleport-read-vm",
+      "metadata": {
+        "description": "Friendly name of the role definition"
+      }
+    },
+    "identityName": {
+      "type": "string",
+      "defaultValue": "teleport-azure",
+      "metadata": {
+        "description": "Name of the managed identity"
+      }
+    }
+  },
+  "variables": {
+    "roleDefName": "[guid(resourceGroup().id, string('Microsoft.Compute/virtualMachines/read'))]"
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Authorization/roleDefinitions",
+      "apiVersion": "2022-04-01",
+      "name": "[variables('roleDefName')]",
+      "properties": {
+        "roleName": "[parameters('roleName')]",
+        "description": "A role to allow reading information about Virtual Machines, to be used for Teleport.",
+        "type": "customRole",
+        "permissions": [
+          {
+            "actions": [
+              "Microsoft.Compute/virtualMachines/read"
+            ],
+            "notActions": []
+          }
+        ],
+        "assignableScopes": [
+          "[resourceGroup().id]"
+        ]
+      }
+    },
+    {
+      "type": "Microsoft.ManagedIdentity/userAssignedIdentities",
+      "name": "[parameters('identityName')]",
+      "apiVersion": "2018-11-30",
+      "location": "[resourceGroup().location]"
+    }
+  ],
+  "outputs": {
+    "principalID": {
+      "type": "string",
+      "value": "[reference(resourceId('Microsoft.ManagedIdentity/userAssignedIdentities', parameters('identityName'))).principalId]"
+    },
+    "clientID": {
+      "type": "string",
+      "value": "[reference(resourceId('Microsoft.ManagedIdentity/userAssignedIdentities', parameters('identityName'))).clientId]"
+    },
+    "roleName": {
+      "type": "string",
+      "value": "[variables('roleDefName')]"
+    }
+  }
+}
+```
+
+Then run the following commands to create the custom role and managed identity:
+
+```code
+$ DEPLOY_OUTPUT=$(az deployment group create \
+--resource-group <your-resource-group> \
+--template-file teleport-create-identity.json)
+$ PRINCIPAL_ID=$(echo $DEPLOY_OUTPUT | jq -r '.properites.outputs.principalID.value')
```

Comment:
I'd add that `jq` needs to be installed. Otherwise, some people are not familiar with it may end up with `PRINCIPAL_ID=jq not found`